kyo-bench/src/main/scala/kyo/bench/ForeachBench.scala

```diff
@@ -0,0 +1,61 @@
+package kyo.bench
+
+import kyo.*
+import org.openjdk.jmh.annotations.*
+import org.openjdk.jmh.annotations.Param
+import org.openjdk.jmh.infra.Blackhole
+import scala.compiletime.uninitialized
+import scala.util.Random
+
+class ForeachBench extends BaseBench:
+    @Param(Array("1024", "65536", "262144", "1048576"))
+    var size: Int = uninitialized
+
+    var intChunk: Chunk[Int] = uninitialized
+    var intVec: Vector[Int]  = uninitialized
+
+    def intFillFn: Int = Random.nextInt()
+
+    def f(i: Int): Int < Any = i + 1
+
+    @Setup(Level.Trial)
+    def setup(): Unit =
+        intChunk = Chunk.fill(size)(intFillFn)
+        intVec = intChunk.toVector
+    end setup
+
+    @Benchmark def kyoForeach(bh: Blackhole) =
+        val prg = Kyo.foreach(intChunk)(f).map(_.size)
+        bh.consume(prg.eval)
```

Comment:
The return values of benchmark methods are automatically consumed by a blackhole. There's no need for the explicit `Blackhole` use